HELPS/ClientApp/src/components/user/User.tsx

```diff
@@ -33,8 +53,24 @@ class User extends Component<UserProps> {
                 <div className='col m-3'>
                     {this.getUserDetails()}
                 </div>
+                <div>
+                    <input
+                        type="Number"
+                        value={this.state.searchTerm}
+                        ref="Search"
+                        onChange={(e) => console.log(e.target.value)}
+                        placeholder="Search..."
+                    />
+                </div>
+                <ul>
+                    {this.loadAllUsers.map(student)}
```

Comment:
loadAllUsers is a prop not a local member so you could call it like this.props.loadAllUsers() but you should definitely not be calling this in the render function. Doing so would cause a network request every time the user made a change (e.g. if the user starts typing in the name field it would make a new network request for every letter)